CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 
 
+Breaking changes
+
+- Unify all message paths to follow pattern `<package>/<message_name>`
```

Comment:
Curious if it is actually a breaking change. It's not a public api, so it does not break or change anything for the client?

Reply:
:thinking:

Reply:
The only place it is "breaking" is in the genesis configuration of `msgfee`, as well as queries on `msgfee`.